communication/marketing-team/social-guidelines.md

```diff
@@ -1,23 +1,39 @@
-Future home of social media guidelines
-
-**Twitter:** K8sContributor
-
-Examples:
-1. Try and tweet at least twice a week
-2. No more than two original tweets a day, unlimited retweets; responding to others does not count in this  
-3. should respond within 48 hours
-4. Contributor information/news/stories is the target; forward others to social@cncf.io for @kubernetesio main twitter account  
-5. Bio should mention that this is powered by sig-contribex, link to the kubernetes/community repo  
-6. Never include someones photo, twitter handle, or other personal identifiable info without their explicit permission  
-7. try not to abbreviate as much as possible  
-8. all event tweets should have an action and/or thumbnail image    
-
-Themes:
-- contributor/sig blog posts
-- events
-- keps
-- process changes
-- release info, code freeze, et al
-
-### open-ended questions:
-who do we follow / does it matter  
+# Social media guidelines
+
+**Twitter:** [K8sContributor](https://twitter.com/k8scontributors)
+
+## General guidelines
+
+All messaging must be consistent with the values and principles of the project and the [ethos/vision of the team](https://github.com/kubernetes/community/blob/master/communication/marketing-team/CHARTER.md#ethosvision). Messaging should be positive and uplifting. Be aware that sarcasm and/or jokes are generally hard to read over a medium such as social media.
+
+Original messages should come out of this teams' accounts at least twice a week (if not more). However, there should be no more than three original messages per day. Retweets, likes, and responses are unlimited but, they should be thoughtful. All event messages should have an action and/or thumbnail image (click link for details, register today, etc. are example actions). Never include photos, user handles, or other personally identifiable information without explicit permission.
```

Comment:
Minor recommendation -- the only limit I find worth setting explicitly is _tweets per hour_. Said another way, if we have enough to share where we can add something worthwhile every hour in the day, that'd be wonderful. If you agree, here's a suggestion: 

```suggestion
Original messages should come out of this teams' accounts at least twice a week (if not more). However, there should be no more than three tweets per hour (including retweets). Retweets, likes, and responses are unlimited, but they should be used thoughtfully to encourage inclusive and kind participation. Use scheduling software as needed to space out messages. All messages should serve a purpose: have an action and/or thumbnail image ("click link for details," "register today," etc. are example actions). Never include photos, user handles, or other personally identifiable information without explicit permission.
```